pkg/vpcmodel/textOutput.go

```diff
@@ -15,7 +15,8 @@ type TextOutputFormatter struct {
 }
 
 func multipleVPCsConfigHeader(c *VPCConfig) (string, error) {
-	if len(c.RoutingResources) != 1 {
+	if len(c.RoutingResources) != 2 { // expecting tgw and sgw (virtual gateway)
```

Comment:
same here

Reply:
aws does not support multi-vpc

Reply:
We may want to support multi-vpc in AWS in the future.
@adisos , any insight here?

Reply:
I would change this check here as well, to check here that `c.RoutingResources` contains exactly one router of kind TGW for `MultipleVPCsConfig`.
If we support multiple VPCs for AWS as well, this is a more consistent check, and does not depend on whether or not service network gateway is present.

Reply:
fixed